compass/ocean/tests/internal_wave/namelist.forward

```diff
@@ -0,0 +1,10 @@
+config_dt = '00:05:00'
+config_btr_dt = '00:00:15'
+config_time_integrator = 'split_explicit'
+config_run_duration = '0000_00:15:00'
+config_use_cvmix = .true.
+config_pio_stride = 4
+config_use_mom_del2 = .true.
+config_mom_del2 = 10.0
+config_implicit_bottom_drag_coeff = 1.0e-2
+config_use_cvmix_convection = .false.
```

Comment:
@mark-petersen, you pushed this change as part of your commit.  I want to make sure it was intentional, since it wasn't mentioned in the commit message.

In general, I think it's a better practice to suggest changes as part of a review rather than pushing them yourself.

Reply:
You are right, that was unintentional. See comment below, I was testing that at the same time. I just corrected it, but I'll do the `suggest changes` method next time.